system_test/common/helpers/aest_docker.erl

```diff
@@ -286,7 +292,8 @@ setup_node(Spec, BackendState) ->
             {rw, LogPath, ?EPOCH_LOG_FOLDER}] ++
             [ {ro, Genesis, ?EPOCH_GENESIS_FILE} || Genesis =/= undefined ] ++
             [ {rw, element(1, DbPath), element(2, DbPath)} || DbPath =/= undefined ],
-        ports => PortMapping
+        ports => PortMapping,
+        entrypoint => maps:get(entrypoint, Spec, ["/docker-entrypoint.sh"])
```

Comment:
If no `entrypoint` in the spec, please do not specify the `entrypoint` key at all.

Specifying the default value `["/docker-entrypoint.sh"]` couples further the test framework to the Docker image version.

Specifying the default value `[""]` would not couple further the test framework to the Docker image version but would couple further the test framework to the Docker engine API version.

----
I see underlying API is [this](https://docs.docker.com/engine/api/v1.39/#operation/ContainerCreate):
> Entrypoint 
> Array of string
> The entry point for the container as a string or an array of strings.
> If the array consists of exactly one empty string (`[""]`) then the entry point is reset to system default (i.e., the entry point used by docker when there is no `ENTRYPOINT` instruction in the `Dockerfile`).

Reply:
Entrypoint is not passed to the API call at all if not set explicitely. Default is removed from here.